readme.md

```diff
@@ -53,7 +55,7 @@ console.log(results);
 ```
 
 Your Answer:
-```text
+```var results = Author.name.find({"Bob"});
 
 ```
```

Comment:
The issue here isn't that Mongoose can't find an author with the name "bob". The issue is that `.find` is an asynchronous method so, in the current code, `results` is undefined because `.find` is not done running yet. Instead, we need to make use of a callback like so...

```js
var results = AuthorModel.find({name: "Bob"}, function(results){
  console.log(results);
});
```